Dockerfile

```diff
@@ -29,7 +29,8 @@ ENV  BASH_PROFILE=".bashrc" \
      CLIENT_MAX_BODY_SIZE="75M"
 
 # Tethys settings arguments
-ENV  ALLOWED_HOSTS="localhost 127.0.0.1" \
+ENV  DEBUG="True" \
```

Comment:
I see the Dockerfile being primarily used for production, so I would recommend having the default values be appropriate for a production deployment if possible (i.e.: DEBUG="False").

Reply:
Yes, that was my intention. Oops!